packages/wonder-blocks-core/components/clickable-behavior.js

```diff
@@ -63,17 +63,17 @@ type State = {
 };
 
 export type ClickableHandlers = {
-    onClick: (e: SyntheticMouseEvent<>) => void,
-    onMouseEnter: () => void,
-    onMouseLeave: () => void,
-    onMouseDown: () => void,
-    onMouseUp: (e: SyntheticMouseEvent<>) => void,
-    onTouchStart: () => void,
-    onTouchEnd: () => void,
-    onTouchCancel: () => void,
-    onKeyDown: (e: SyntheticKeyboardEvent<*>) => void,
-    onKeyUp: (e: SyntheticKeyboardEvent<*>) => void,
-    onBlur: (e: SyntheticFocusEvent<*>) => void,
+    onClick?: (e: SyntheticMouseEvent<>) => void,
+    onMouseEnter?: () => void,
+    onMouseLeave?: () => void,
+    onMouseDown?: () => void,
+    onMouseUp?: (e: SyntheticMouseEvent<>) => void,
+    onTouchStart?: () => void,
+    onTouchEnd?: () => void,
+    onTouchCancel?: () => void,
+    onKeyDown?: (e: SyntheticKeyboardEvent<*>) => void,
+    onKeyUp?: (e: SyntheticKeyboardEvent<*>) => void,
+    onBlur?: (e: SyntheticFocusEvent<*>) => void,
```

Comment:
These handlers aren't set in docs.md for `ButtonCore`.

Reply:
This also doesn't feel like something we want to have be optional - we really want all of these to be defined every time. It might be better to create placeholder handlers for these and stick them in docs.md (so we can do something like `{...defaultHandlers}`, perhaps).

Reply:
Whoops... thanks for catching this.